src/main/java/com/google/sps/data/DeliverySlot.java

```diff
@@ -14,29 +14,94 @@
 package com.google.sps.data;
 import com.google.gson.Gson;
 import com.google.gson.GsonBuilder;
-import java.util.Date; 
+import com.google.maps.errors.ApiException;
+import com.google.maps.model.LatLng;
+
+import java.io.IOException;
+import java.util.Date;
 import java.time.LocalTime;
 
-/** 
+/**
  * Class that represents a delivery timeslot using delivery date, start and end times.
  */
 public class DeliverySlot {
+  /** 
+   * Enum with the properties of a delivery slot in datastore.
+   */
+  public enum Property {
+    START_TIME("starTime"),
+    END_TIME("endTime"),
+    START_LAT("startLat"),
+    START_LNG("startLng"),
+    USER_ID("uid");
+
+    public final String label;
+
+    private Property(String label) {
+      this.label = label;
+    }
+  }
+
   private Date startTime;
```

Comment:
Please add comments to what these values represent.
e.g. "The time that a courier starts their delivery run."

I understand how that might be obvious for you, but it might not obvious for all readers.

Reply:
Done.